### PR TITLE
[1LP][RFR] Adding ToggleFullScreen Button Test for HTML5 Remote Console

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Adding tests__ for feature ToggleFullScreen Button on HTML5 Remote Console. 

**Expected Result:**Tests are supposed to fail against the vsphere6-nested and RHV41 because it is a known Bug (https://bugzilla.redhat.com/1479935) and RHOS7-GA would be skipped for ToggleFullScreen button test since it does not support this functionality.  

{{ pytest: cfme/tests/cloud_infra_common/test_html5_vm_console.py --use-provider vsphere6-nested --use-provider default }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 670b1234fab2fa5dba1458209b89dd22a33ff85e:

`cfme/common/vm_console.py`:
- :red_circle: Line 17:1: **F401** *'selenium.common.exceptions.NoSuchElementException' imported but unused*
- :red_circle: Line 18:1: **F811** *redefinition of unused 'Keys' from line 13*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: requirements/frozen.txt [hunk 1]
```diff
@@ -199,4 +199,4 @@ wrapanapi==2.3.1
 wrapt==1.10.10
 xmltodict==0.11.0
 yaycl==0.2.0
-yaycl-crypt==0.2.0
+yaycl-crypt==0.2.0
```
Comment (quarckster) [inline requirements/frozen.txt]: This dependency already exists in [frozen.txt](https://github.com/ManageIQ/integration_tests/blob/master/requirements/frozen.txt#L202)

File: cfme/tests/cloud_infra_common/test_html5_vm_console.py [hunk 2]
```diff
@@ -181,6 +181,10 @@ def _validate_login():
         assert vm_console.wait_for_text(text_to_find="login:", timeout=200),\
             "VM Console didn't prompt for Login"
 
+        if not provider.one_of(OpenStackProvider):
+            assert vm_console.send_fullscreen(), "VM Console Toggle Full Screen button does \
+             not work"
```
Comment (quarckster) [inline cfme/tests/cloud_infra_common/test_html5_vm_console.py]: **REQUIRED**
Please use `()` to break lines.

**Comment by kedark3** [comment]
@quarckster : Requested changes are made :)

File: requirements/frozen.txt (hunk 1 above)
Comment (kedark3) [inline requirements/frozen.txt]: Added this file by mistake

**Comment by kedark3** [comment]
As expected, test against the vsphere6-nested and RHV41 is failing, because of a known bug mentioned above in description, and RHOS7-GA has some issue mentioned in #5190

**Pull request merged**